lib/appium.js

```diff
@@ -155,7 +155,8 @@ class AppiumDriver extends BaseDriver {
 
       const {
         driver: InnerDriver,
-        version: driverVersion
+        version: driverVersion,
+        driverName: driverName
```

Comment:
when we have `key: key` we can replace it with just `key`, so this can be:

```js
const {
  driver: InnerDriver,
  version: driverVersion,
  driverName
} = ...
```